packages/storybook-a11y-test/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
 
+## Unreleased
+
+### Changed
+
+- Update storybook addon deps and remove use of fromId() in favor of loadStory(), introduces class based approach. [[#2194](https://github.com/Shopify/quilt/pull/2194)]
```

Comment:
```suggestion
- Now requires storybook >6.4.0. Add support for storyStoreV7 mode. Reuse open browser instances instead of opening new browsers every time, which reduces memory pressure and makes the tests faster. [[#2194](https://github.com/Shopify/quilt/pull/2194)]
```